frontend/src/views/Transfers/AddTransfer/components/SigningAuthority.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Box, FormControlLabel, Checkbox, Typography } from '@mui/material'
+import LabelBox from './LabelBox' // Assuming LabelBox is already defined in your project
+import BCTypography from '@/components/BCTypography'
+import withRole from '@/utils/withRole'
+
+const SigningAuthority = ({ register, errors }) => {
+  return (
+    <>
+      <BCTypography variant={'h6'} mt={2} mb={2} color={'primary'}>
```

Comment:
To be consistent with other components we can define the header inside LabelBox.
    <LabelBox
      label="Signing Authority Declaration"
    >

Reply:
the wireframe has a different styling for Signing Authority so that's why we left it out.